cmd/snap-mgmt/snap-mgmt.sh.in

```diff
@@ -97,6 +97,15 @@ purge() {
                         rmdir --ignore-fail-on-non-empty "$d"
                     fi
                 done
+                # udev rules
+                find /etc/udev/rules.d -name "*-snap.${snap}.rules" -print0 | xargs -0 rm -f
```

Comment:
Currently we always use `70-snap.*.rules` but this is fine. I wonder if we should also look inside but since snapd won't look inside such files (to look for any markers) I think this is okay.

The rule can be simplified slightly with: `find /etc/udev/rules.d -name "*-snap.*.rules" -exec rm -f "{}" \;`

Reply:
I like the find|xargs pattern slightly better because it avoids unneeded fork/exec but no super strong opinion.

Reply:
I'll do the `.. -exec` change, but I'd rather stay with snap specific match than a catchall one.